spec/js/views/app_view_spec.js

```diff
@@ -18,6 +18,42 @@ define([
     });
   });
 
+
+  describe('addTodo', function(){
+    beforeEach(function(){
+      subject = function () {
+        instance.addTodo();
+      };
+    });
+    
+    describe('with empty title', function(){
+      it('will not add todo', function(){
+        subject();
+        expect(instance.todoItemCollection.length).toEqual(0);
+      });
+    });
+
+    describe('with some title', function(){
+      beforeEach(function(){
+        instance.$("#todo-title-textbox").val("test");
+        subject();
```

Comment:
Why calling the subject in `beforeEach` ? How would you future refine the setup for each test / other contexts?